win32/updateBuildLibs.py

```diff
@@ -19,7 +19,7 @@
 CURRENT_PATH = os.getcwd()
 # save the build log in the output directory
 LOG_PATH = os.path.join(CURRENT_PATH, 'output', time.strftime("%Y.%m.%d-%H.%M.%S"))
-
+APPVEYOR = os.getenv("APPVEYOR",False)
```

Comment:
I'd rather not have any APPVEYOR specific stuff in here.   I think we should change this such that there is an argument that people can pass in so that only 64-bit Release_PostgreSQL is built.